src/labels/CustomLabels.labels

```diff
@@ -1721,6 +1721,14 @@
         <shortDescription>Checks if Elevate Recurring Donation&apos;s Effective Date is valid or not</shortDescription>
         <value>Please enter a new Effective Date. The Effective Date for an Elevate Recurring Donations cannot be in the future.</value>
     </labels>
+    <labels>
+        <fullName>RD2_ElevateRDCampaignCannotUpdateToNull</fullName>
+        <categories>Recurring-Donations</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Checks if Elevate Recurring Donation&apos;s Campaign update is valid or not</shortDescription>
+        <value>Please enter a Campaign. The Campagin for an Elevate Recurring Donations cannot update to null.</value>
+    </labels>
```

Comment:
@mattwashington @sam-knox not sure who to tag, but can you review the new label.

Reply:
My suggestion: "Please enter a Campaign. Once entered, the Campaign for an Elevate Recurring Donation can be changed, but not removed."

Reply:
Here's what I'd suggest:

"You can't remove the Campaign on an active Elevate-connected Recurring Donation. To remove the Campaign, close the Recurring Donation and create a new one."